recipes/cppcommon/all/conanfile.py

```diff
@@ -36,64 +41,71 @@ def _compilers_minimum_version(self):
             "clang": 6,
             "gcc": 7,
             "Visual Studio": 16,
+            "msvc": 192,
         }
 
+    def export_sources(self):
+        export_conandata_patches(self)
+
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.compiler == "Visual Studio" and self.settings.arch == "x86":
-            raise ConanInvalidConfiguration("Visual Studio x86 builds are not supported.")
-
         if self.options.shared:
-            del self.options.fPIC
-
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, "17")
+            self.options.rm_safe("fPIC")
 
-        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
-        if minimum_version:
-            if tools.Version(self.settings.compiler.version) < minimum_version:
-                raise ConanInvalidConfiguration("cppcommon requires C++17, which your compiler does not support.")
-        else:
-            self.output.warn("cppcommon requires C++17. Your compiler is unknown. Assuming it supports C++17.")
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("fmt/8.0.0")
+        if Version(self.version) < "1.0.3" or self.version == "cci.20201104":
+            self.requires("fmt/8.1.1")
+        else:
+            self.requires("fmt/10.0.0")
         if self.settings.os == "Linux":
             self.requires("libuuid/1.0.3")
 
```

Comment:
Seems like it needs a cmake tool_requires?

Reply:
It is a few lines below?

Reply:
@memsharded 
Thank you for your advise.
You are right.
I fixed it.